doc/cephadm/install.rst

```diff
@@ -369,3 +369,6 @@ RADOS pool *nfs-ganesha* and namespace *nfs-ns*,::
 
 See :ref:`orchestrator-cli-placement-spec` for details of the placement specification.
 
+Deploying by means of containers
+================================
+It is also possible to deploy Ceph by means of containers. See :ref:`containers` for information on how to deploy Ceph by means of containers.
```

Comment:
This sentence sounds wrong somehow. cephadm **always** deploys ceph using containers. It's just that cephadm allows you to use different containers. 

The Ceph project provides a lot of different containers: We're providing the `ceph/ceph`  image, `ceph/daemon-base` image etc. in different versions with different tags. All of those different containers provided by Ceph are described in  doc/install/containers.rst

The warning to not use `:latest` is on the other hand related to cephadm and rook and is not related to the containers provided by the Ceph project.

Reply:
I've fuzzed up the text a bit so that it merely directs the reader to the containers.rst page, without committing the text to any incorrect claims.